server/pulp/server/webservices/views/repo_groups.py

```diff
@@ -10,7 +10,7 @@
 from pulp.server.managers import factory as managers_factory
 from pulp.server.managers.repo.group import query as repo_group_query
 from pulp.server.managers.repo.group.publish import publish as repo_group_publish
-from pulp.server.webservices.controllers.decorators import auth_required
+from pulp.server.webservices.views.decorators import auth_required
```

Comment:
This line is out of order with the rest of the imports for alphabetical reasons.